nx-X11/programs/Xserver/os/Imakefile

```diff
@@ -75,10 +75,17 @@ XDMAUTHOBJS =
 XDMAUTHSCRS =
 #endif
 
+TIRPC_INCLUDES =
```

Comment:
Move further down into a 
```
#if UseTIRPC
... 
#else
<here>
#endif
```

Reply:
The intent was to clear the variable in all cases, also when `HasSecureRPC` is false.

If moving that into these conditionals, I'd need to do it two times when an added `#else` branch since the rest of the code also sets or clears the vars based on conditionals. IMHO the current solution provides a smaller and more elegant diff.